include/zeda/zeda_misc.h

```diff
@@ -212,7 +212,9 @@ __EXPORT int atox(char *str);
  * itoa() does not check the size of \a buf.
  * It assumes that \a 'buf' has an enough size.
  */
+#ifndef __WINDOWS__
```

Comment:
- Do not use `__WINDOWS__` here as commented above.
 - itoa() is an obsolete function. It was renamed to zI2A().